tests/queries/0_stateless/03533_column_array_insert_broken_offsets.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# Refs: https://github.com/ClickHouse/ClickHouse/issues/81199
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT 'create table array_deserialize_offsets (val Array(String)) engine=Memory()'
+
+{
+  # Native header
+  chc "select * from values('val Array(String)', (['foo','fooo']),(['bar','barr'])) format Native" | head -c20
```

Comment:
I don't think we can use `chc` in tests since it's missing client config (or might not be in the PATH). In any case it might be better to just use `${CLICKHOUSE_LOCAL}` instead